.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,50 @@
+// For more details, see https://aka.ms/devcontainer.json.
+{
+  "name": "P24 Pay",
+  "workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}/",
+  "image": "mcr.microsoft.com/devcontainers/javascript-node:20",
+  "postCreateCommand": "npm install --ignore-scripts",
+  "postAttachCommand": {
+    "Start server": "CI=true npx kill-port --yes 8080 && npm start"
+  },
+  "forwardPorts": [
+    8080
+  ],
+  "portsAttributes": {
+    "8080": {
+      "label": "Preview of P24 Pay Flow",
+      "onAutoForward": "openBrowser"
+    }
+  },
+  "secrets": {
+    "CLIENT_ID": {
+      "description": "Sandbox client ID of the application.",
+      "documentationUrl": "https://developer.paypal.com/api/rest/"
+    },
+    "CLIENT_SECRET": {
+      "description": "Sandbox secret of the application.",
+      "documentationUrl": "https://developer.paypal.com/api/rest/"
+    },
+    "WEBHOOK_ID": {
+      "description": "Sandbox Webhook ID of the application.",
+      "documentationUrl": "https://developer.paypal.com/api/rest/webhooks/"
+    },
+    "PAYPAL_API_BASE": {
+      "description": "Sandbox PayPal API base url of the application.",
+      "documentationUrl": "https://api.sandbox.paypal.com/"
```

Comment:
We should not put this url in documentationUrl field as users might see it as documentation location, and when they will visit this page, it would return 404.
is there any way to set this url as default value in this secret? if not, we should put this in description and remove documentationUrl.

![image](https://github.com/paypal-examples/p24/assets/141275730/58b86b3a-013c-499c-b45e-ddb8ea3effee)